src/request/sdata.rs

```diff
@@ -46,25 +46,21 @@ pub enum SDataRequest {
     },
     /// Get last entry from the Sequence.
     GetLastEntry(SDataAddress),
-    /// List all current users permissions.
-    GetPermissions(SDataAddress),
+    /// Get the current access policy.
+    GetPolicy(SDataAddress),
     /// Get current permissions for a specified user(s).
-    GetUserPermissions {
+    GetPermissions {
         /// Sequence address.
         address: SDataAddress,
         /// User to get permissions for.
         user: SDataUser,
     },
-    /// Get current owner.
-    GetOwner(SDataAddress),
-    /// Set new permissions for public Sequence.
-    MutatePubPermissions(SDataMutationOperation<SDataPubPermissions>),
-    /// Set new permissions for private Sequence.
-    MutatePrivPermissions(SDataMutationOperation<SDataPrivPermissions>),
-    /// Add a new `owners` entry. Only the current owner(s) can perform this action.
-    MutateOwner(SDataMutationOperation<SDataOwner>),
+    /// Set new policy for public Sequence.
+    MutatePubPolicy(SDataDataMutationOp<SDataPubPolicy>),
```

Comment:
Could you explain why the `MutatePubPolicy` and `MutatePrivPolicy` variants are backed by `SDataDataMutationOp<T>`? Should  they be `SDataPolicyMutationOp<T>` instead, or am I missing something?

Reply:
It could be yes, it's just the since it's WIP and still didn't finish the part of defining the policy ops so it's not properly reflected there.

Reply:
That makes sense, thanks for the clarification 👍